boefjes/boefjes/plugins/kat_answer_parser/normalize.py

```diff
@@ -18,3 +18,13 @@ def run(normalizer_meta: NormalizerMeta, raw: bytes | str) -> Iterable[OOI]:
             bit_id="port-classification-ip",
             config=json.loads(raw),
         )
+
+    if "/bit/disallowed-csp-hostnames" in mime_types:
```

Comment:
That leading / is weird here. I think we should fix that for all question outputs.

Reply:
Will do! Creating questions is also a bit of a pain I noticed. Took me a while. We should revisit that when the time comes.

Reply:
When you have the Json you want (as in, this json contains all the data We need), you can use something like https://www.jsonschema.net/ to create a version of the jsonschema/question that would produce such a json.

Reply:
Yeah but I mean that you have to add a bit that creates the question and then add it to the answer parser just feels a bit redundant. Maybe we could have one bit that adds all the questions on network for example.

Reply:
Checked code and leading / has several uses. Cant remove for now.